src/cli/fuzzer_bridge/models.py

```diff
@@ -0,0 +1,140 @@
+
+    chainId: HexNumber = Field(..., alias="chainId")
+    address: Address
+    nonce: HexNumber
+    v: HexNumber  # yParity
+    r: HexNumber
+    s: HexNumber
+
+    class Config:
+        """Pydantic configuration."""
+
+        populate_by_name = True
+
+
+class FuzzerTransaction(BaseModel):
+    """Transaction definition in fuzzer output."""
+
+    from_: Address = Field(..., alias="from")
+    to: Optional[Address] = None
+    value: HexNumber = HexNumber(0)
+    gas: HexNumber
+    gasPrice: Optional[HexNumber] = Field(None, alias="gasPrice")
+    maxFeePerGas: Optional[HexNumber] = Field(None, alias="maxFeePerGas")
+    maxPriorityFeePerGas: Optional[HexNumber] = Field(None, alias="maxPriorityFeePerGas")
+    nonce: HexNumber
+    data: Bytes = Bytes(b"")
+    accessList: Optional[List[Dict[str, Union[str, List[str]]]]] = Field(None, alias="accessList")
+    blobVersionedHashes: Optional[List[Hash]] = Field(None, alias="blobVersionedHashes")
+    maxFeePerBlobGas: Optional[HexNumber] = Field(None, alias="maxFeePerBlobGas")
+    authorizationList: Optional[List[FuzzerAuthorization]] = Field(None, alias="authorizationList")
+
+    class Config:
+        """Pydantic configuration."""
+
+        populate_by_name = True
+
+
+class FuzzerEnvironment(BaseModel):
+    """Environment definition in fuzzer output."""
+
+    currentCoinbase: Address = Field(..., alias="currentCoinbase")
+    currentDifficulty: HexNumber = Field(HexNumber(0), alias="currentDifficulty")
+    currentGasLimit: HexNumber = Field(..., alias="currentGasLimit")
+    currentNumber: HexNumber = Field(..., alias="currentNumber")
+    currentTimestamp: HexNumber = Field(..., alias="currentTimestamp")
+    currentBaseFee: Optional[HexNumber] = Field(None, alias="currentBaseFee")
+    currentRandom: Optional[Hash] = Field(None, alias="currentRandom")
+    currentExcessBlobGas: Optional[HexNumber] = Field(None, alias="currentExcessBlobGas")
+    currentBlobGasUsed: Optional[HexNumber] = Field(None, alias="currentBlobGasUsed")
+    parentBeaconBlockRoot: Optional[Hash] = Field(None, alias="parentBeaconBlockRoot")
+    parentUncleHash: Optional[Hash] = Field(None, alias="parentUncleHash")
+    parentDifficulty: Optional[HexNumber] = Field(None, alias="parentDifficulty")
+    parentBaseFee: Optional[HexNumber] = Field(None, alias="parentBaseFee")
+    parentGasUsed: Optional[HexNumber] = Field(None, alias="parentGasUsed")
+    parentGasLimit: Optional[HexNumber] = Field(None, alias="parentGasLimit")
+
+    class Config:
+        """Pydantic configuration."""
+
+        populate_by_name = True
+
+
+class FuzzerOutput(BaseModel):
+    """Main fuzzer output format v2."""
+
+    version: str = Field(..., pattern="^2\\.0$")
+    fork: str
+    chainId: int = Field(1, alias="chainId")
+    accounts: Dict[Address, FuzzerAccount]
+    transactions: List[FuzzerTransaction]
+    env: FuzzerEnvironment
+    parentBeaconBlockRoot: Optional[Hash] = Field(None, alias="parentBeaconBlockRoot")
```

Comment:
```suggestion
    parent_beacon_block_root: Hash | None = None
```

Example of the proper name mentioned in previous comment.